assets/js/common/UpgradablePackagesList/UpgradablePackagesList.test.jsx

```diff
@@ -10,7 +10,9 @@ describe('UpgradablePackagesList component', () => {
     const { patches } = upgradablePackage;
 
     const expectedInstalledPackage = `${upgradablePackage.name}-${upgradablePackage.from_version}-${upgradablePackage.from_release}.${upgradablePackage.arch}`;
+    upgradablePackage.installed_package = expectedInstalledPackage;
```

Comment:
nitpick: maybe something like this would look better rather than direct assignment:
```
const enrichedUpgradablePackage = {
    ...upgradablePackage,
    installed_package: expectedInstalledPackage,
    latest_package: expectedLatestPackage,
}
```